### ISSUE TITLE
[Feature Request] Live Updates

### ISSUE THREAD

**Issue by Vastrix** [acaudwell/Gource#195]
Greetings
  
Would it be possible to keep the graph/animation up to date (**live**) with the target repo/custom log

o/

**Comment by mschilli87** [comment]
I would have expected

```sh
gource --start-date "$(date '+%Y-%m-%d %H:%M')" --realtime
```

to do just that but (at least in Gource v0.43) this fails with

```
gource: failed to generate log file for the specified time period
```

:disappointed:

**Comment by Vastrix** [comment]
Ah, Interesting, thought `realtime` meant no time scaling (every second is a second)

**Comment by mschilli87** [comment]
Yes, but it will start at the begining and not *now* which I tried to enforce via the `--start-date` argument.
